index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang='en'>
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		
+		<title>wommy.io - home</title>
+
+		<link rel='stylesheet' href='./style.css'>
+		<style>
+
+		</style>
```

Comment:
What's with the empty style tag here?

Reply:
it will be used soon

Reply:
If you are getting a code review, it should either be removed, used, or contain a note about why it is there and empty